qemu/tests/cfg/block_boot_multi_disks.cfg

```diff
@@ -0,0 +1,44 @@
+- block_boot_multi_disks:
+    type = block_boot_multi_disks
+    start_vm = no
+    vt_ulimit_nofile = 65536
+    only i386, x86_64
+    force_create_image = yes
+    force_create_image_image1 = no
+    remove_image = yes
+    remove_image_image1 = no
+
+    stg_image_size = 512M
+    stg_image_format = qcow2
+    stg_image_name = "images/%s"
+    stg_drive_format = virtio
+
+    check_disk_num = 6
+    login_timout = 360
+
+    Linux:
+        tmp_dir = /var/tmp/test
+        check_num_cmd = "lsblk -nd|grep [sv]d|wc -l"
+        guest_cmd = "mkdir -p ${tmp_dir} && mkfs.xfs -f {0}  &&"
+        guest_cmd += " mount -t xfs {0} ${tmp_dir} && dd if=/dev/zero "
+        guest_cmd += " of=${tmp_dir}/test.img bs=1M count=256 oflag=direct && "
+        guest_cmd += " umount ${tmp_dir}"
+    Windows:
+        check_num_cmd = powershell -command (get-disk).count
+        guest_cmd = "WIN_UTILS:\coreutils\DummyCMD.exe {} 102400000 1"
+    variants:
+        - with_default_bios:
+            only default_bios
+            check_message = "WARNING - Unable to allocate resource at add_drive"
```

Comment:
```
i386, x86_64:
    check_message = "WARNING - Unable to allocate resource at add_drive"
```

The parameters here can be limited to x86 only, and I think other platforms should also be able to pass. @maxujun @fbq815 could you try it?

```
 (1/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_64k.io-github-autotest-qemu.block_boot_multi_disks.with_default_bios.with_in_limit.arm64-pci: PASS (52.41 s)
 (2/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_64k.io-github-autotest-qemu.block_boot_multi_disks.with_default_bios.with_over_limit.arm64-pci: PASS (53.86 s)
```

Reply:
aarch64:
                check_message =

I set it as above due to do not knowing other platform behaviors.  i remove the limit only x86_64. and other platform can test it , hi @zhenyzha ,could you help to re-test ,thanks.

Reply:
Yes, this modification is okay for arm. Let us wait for feedback from other platforms.

```
 (1/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_4k.io-github-autotest-qemu.block_boot_multi_disks.with_default_bios.with_in_limit.arm64-pci: STARTED
 (1/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_4k.io-github-autotest-qemu.block_boot_multi_disks.with_default_bios.with_in_limit.arm64-pci: PASS (54.98 s)
 (2/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_4k.io-github-autotest-qemu.block_boot_multi_disks.with_default_bios.with_over_limit.arm64-pci: STARTED
 (2/2) Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_4k.io-github-autotest-qemu.block_boot_multi_disks.with_default_bios.with_over_limit.arm64-pci: PASS (53.25 s)
```

Reply:
since we use s390x-ccw bios instead of seabios on s390x, I think we could skip s390, s390x for this case

Reply:
Same as s390x, and we don't have this block devices limit for power BIOS.

Reply:
there are two solution 
1.
aarch64,s390,s390x:
check_message =
2. filter out s390 ,s390x
no s390,s390x 

@maxujun ,what is your opinion ?

Reply:
No the discussion here is wrong,
When you put check_message =' ' it will skip the bios check and directly boot with multiple hard drives and run the stress.
@PaulYuuu What do you think?

Reply:
Choose 2，because it's not suitable from case title，I mean "in_limit" and "over_limit" for power.we have others multi disks case can cover this case.

Reply:
> No the discussion here is wrong, When you put check_message =' ' it will skip the bios check and directly boot with multiple hard drives and run the stress. @PaulYuuu What do you think?

For other architectures, all logs are saved into serial log I think. So please check the serial log to see if the message exists, if yes, we can use `vm.serial_console` instead.

Reply:
> > No the discussion here is wrong, When you put check_message =' ' it will skip the bios check and directly boot with multiple hard drives and run the stress. @PaulYuuu What do you think?
> 
> For other architectures, all logs are saved into serial log I think. So please check the serial log to see if the message exists, if yes, we can use `vm.serial_console` instead.

@maxujun could you please check your platform to see what happens?

Reply:
No this message and no this disk number limit for power. it's passed if set check_message =' ' ,but I don't think it's significant for these two cases “”with_in_limit“ and ”with_over_limit“，and we have the multi disks case,so needn't similar case for other platform，it‘s only suitable for x86，I think.

Reply:
multi_disk.max_disk case will boot up 24 disks and do io test,so this case can cover this cases,so please filter ppc64le for these two cases  with "no ppc64le".

Reply:
updated